### DISCUSSION TITLE
Forgot email associated with my old account

### DISCUSSION THREAD

**Discussion by KioiJr** [community/community#143434] [Other Feature Feedback, Questions, & Ideas]
Hello,
I have an old GitHub account that I would like to access, but the problem is that I cannot remember the email address I linked it with. A verification code is being sent to that address. Is there a way to go solve this?

**Comment by To-devgoy** [comment]
![Wi-Fi_QR_code_Devilboy](https://github.com/user-attachments/assets/5be7b64c-b5ab-4909-943a-c99942880a85)

**Reply by To-devgoy** [reply]
`open `

**Reply by To-devgoy** [reply]
![Uploading 17306279366515243579502225717661.jpg…]()
_

**Reply by To-devgoy** [reply]
> ![Wi-Fi_QR_code_Devilboy](https://private-user-images.githubusercontent.com/186976537/382561769-5be7b64c-b5ab-4909-943a-c99942880a85.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA2MjgyNzQsIm5iZiI6MTczMDYyNzk3NCwicGF0aCI6Ii8xODY5NzY1MzcvMzgyNTYxNzY5LTViZTdiNjRjLWI1YWItNDkwOS05NDNhLWM5OTk0Mjg4MGE4NS5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwM1QwOTU5MzRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03MWIxYTU5M2ZkYjFkZTE1MWE4ZGI3ODA1YzcwMjlkZGViMGVmZDI1N2YxZjEzZjc0ZGNkODIzYzZkZTgwODQ5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.G5bKc98FY9MBRb-ESOSmPjE5MrrsoCy6itydM8_UIFU)

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @KioiJr ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by KioiJr** [reply]
Thank you, I appreciate it.

**Comment by thomassiard** [comment]
If you can't remember the email linked to your old GitHub account, here are a few steps you can try to regain access:

Check Common Email Accounts: Try logging in with the email addresses you frequently used at the time of creating that GitHub account. GitHub typically allows login with either a username or email address, so experimenting with familiar addresses may help.

Search Your Email Accounts for GitHub Notifications: If you have multiple email accounts, try searching for GitHub-related emails, such as verification emails, notifications, or any old account-related emails that might show which address was used.

Use GitHub’s Support: If you still can’t find the email address, contact GitHub Support directly:

Go to the [GitHub Support page](https://support.github.com/) and explain the situation.
Provide as much information as possible to verify your identity (e.g., possible usernames, associated repository names, any payment information if applicable, etc.).
Two-Factor Authentication (2FA) or Backup Codes: If you set up two-factor authentication (2FA) with the account and still have access to your authenticator app or backup codes, you may be able to bypass the email verification step.

Alternative Recovery: If the account was linked to a social media login, such as Google or Facebook, you might be able to use that option on the login screen.

**Reply by ebndev** [reply]
Hi @thomassiard, thanks for participating in GitHub Discussions!

Please note: we’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  

You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. 

Please feel encouraged to continue engaging in Discussions as your authentic self though. We appreciate you being a part of the Community 🌟
